### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/config/i18n.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#799] [Announcements]
# https://theme-hope.vuejs.press/zh/config/i18n.html

设置语言 你需要为每个语言设置 lang 选项。即使你只在使用单个语言，你也必须在 .vuepress/config.{js,ts} 中设置 lang。 为什么要这样做? 要提供正确的语言环境文本，主题需要知道根文件夹以及每个多语言文件夹正在使用哪种语言。 多语言配置 locales 是一个对象，其键为每个语言的路径前缀，值为该语言的配置，可以包含 t...

https://theme-hope.vuejs.press/zh/config/i18n.html

**Comment by zerowxt** [comment]
重写了插槽，如何进行多语言配置

**Reply by Mister-Hope** [reply]
你看一下源组件是怎么获得 i18n 的，你也照做就好了。
